src/script/conversation/ConversationVerificationStateHandler.ts

```diff
@@ -175,7 +178,12 @@ export class ConversationVerificationStateHandler {
       }
 
       const currentTimestamp = this.serverTimeHandler.toServerTimestamp();
-      const event = z.conversation.EventBuilder.buildDegraded(conversationEntity, userIds, type, currentTimestamp);
+      const event = window.z.conversation.EventBuilder.buildDegraded(
```

Comment:
Window namespace

Reply:
When I remove the namespace here, the tests fail.